version/news/NEWS-2025.04.0-mariposa-orchid.md

```diff
@@ -0,0 +1,22 @@
+## RStudio 2025.04.0 "Mariposa Orchid" Release Notes
```

Comment:
Wrong file. News for the in-development release goes in rstudio/NEWS.md, not this file, which shouldn't be created until we branched for release in a couple months.